src/transformers/modeling_tf_utils.py

```diff
@@ -236,15 +236,15 @@ def compute_loss(self, labels, logits):
         return loss_fn(next_sentence_label, next_sentence_reduced_logits)
 
 
-def detect_tf_missing_unexpected_layers(model, resolved_archive_file):
+def load_tf_weights(model, resolved_archive_file):
     """
-    Detect missing and unexpected layers.
+    Detect missing and unexpected layers
```

Comment:
The docstring should be adapted since this function loads the TF weights **and** detects missing and unexpected layers.